packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.props

```diff
@@ -205,5 +205,5 @@
     <SupportedTargetFramework Remove="@(_UnsupportedNETCoreAppTargetFramework);@(_UnsupportedNETStandardTargetFramework);@(_UnsupportedNETFrameworkTargetFramework)" />
   </ItemGroup>
 
-  <Import Project="Microsoft.WinFX.props" />
+  <Import Project="Microsoft.WinFx.props" />
```

Comment:
The original casing is `WinFX`, so, rename `WinFx` to `WinFX` instead of other way around!